store/sqlstore/compliance_store.go

```diff
@@ -36,19 +37,19 @@ func newSqlComplianceStore(sqlStore SqlStore) store.ComplianceStore {
 func (s SqlComplianceStore) createIndexesIfNotExists() {
 }
 
-func (s SqlComplianceStore) Save(compliance *model.Compliance) (*model.Compliance, *model.AppError) {
+func (s SqlComplianceStore) Save(compliance *model.Compliance) (*model.Compliance, error) {
 	compliance.PreSave()
 	if err := compliance.IsValid(); err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "error validating compliance")
 	}
 
 	if err := s.GetMaster().Insert(compliance); err != nil {
-		return nil, model.NewAppError("SqlComplianceStore.Save", "store.sql_compliance.save.saving.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrap(err, "error saving compliance")
 	}
 	return compliance, nil
 }
 
-func (s SqlComplianceStore) Update(compliance *model.Compliance) (*model.Compliance, *model.AppError) {
+func (s SqlComplianceStore) Update(compliance *model.Compliance) (*model.Compliance, error) {
 	if err := compliance.IsValid(); err != nil {
 		return nil, err
 	}
```

Comment:
You are returning a couple of line later a `model.AppError` should be a regular error wrapped with `errors.Wrap`